eng/psakefile.ps1

```diff
@@ -150,6 +150,7 @@ function Install-LlvmFromSource {
     $Env:PKG_NAME = Get-PackageName
     $Env:CMAKE_INSTALL_PREFIX = $packagePath
     $Env:INSTALL_LLVM_PACKAGE = $true
+    Assert $false -failureMessage "TODO: Migration in progress"
```

Comment:
Is the TODO recorded in the issues?

Reply:
It is being tracked and worked on in the background.